unrejector/views/payment.erb

```diff
@@ -55,19 +55,26 @@
 
     <div class="section red text-center">
       <h1 class="open-sans">Application payment</h1>
+
       <div class="row">
-        <div class="small-10 small-centered large-4 large-centered columns">
+        <% if @paid %>
+          <div class="small-centered columns">
```

Comment:
Does this need a small-centered without a small size? I thought it would default to small-12